mne/datasets/utils.py

```diff
@@ -167,14 +167,14 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
     path = _get_path(path, key, name)
     # To update the testing or misc dataset, push commits, then make a new
     # release on GitHub. Then update the "releases" variable:
-    releases = dict(testing='0.17', misc='0.1')
+    releases = dict(testing='0.18', misc='0.1')
     # And also update the "hashes['testing']" variable below.
 
     # To update any other dataset, update the data archive itself (upload
     # an updated version) and update the hash.
     archive_names = dict(
         misc='mne-misc-data-%s.tar.gz' % releases['misc'],
-        sample='MNE-sample-data-processed.tar.gz',
+        sample='MNE-sample-data-processed-1.tar.gz',
```

Comment:
> Otherwise all the travis builds in other PRs would be failing at the moment.

Only the `master` builds would fail, right? The other PRs shouldn't use `sample` because they don't build doc examples